README.md

```diff
@@ -33,6 +33,24 @@ Use `Redirection#unlink`:
 This will destroy the `Redirection` and re-link its ring neighbors, sealing the
 breach.
 
+## Blacklisting
+
+When blacklisting a URL, specify the bare URL, without `http` or `www`
```

Comment:
This is a bit confusing (it breaks if you blacklist `https://evil.com`), so I documented it. We could also build a `BlacklistCreationService` or something, but...that's a lot of ceremony for something we're going to do very rarely.